wording in intersections (#3017)
dbt-labs/docs.getdbt.com: website/docs/reference/node-selection/set-operators.md

### Intersections

> If multiple arguments to `--select`, `--exclude`, and `--selector` are comma-separated (with no whitespace in between),

Author: ```suggestion
Multiple arguments to `--select`, `--exclude`, and `--selector` can be comma-separated (with no whitespace in between),
```
alternatively

> dbt will select only resources which satisfy _all_ arguments.

Reviewer: ```suggestion

```

### Intersections

> Multiple arguments to `--select`, `--exclude`, and `--selector` can be comma-separated (with no whitespace in between),

Reviewer: great suggestion! i made a small tweak to clarify the language further!

```suggestion
If you separate multiple arguments for `--select`, `--exclude`, and `--selector` with commas and no whitespace in between, dbt will select only resources that satisfy _all_ arguments.
```
